### PR TITLE
add default terms

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/authorization/Authorization.API/API/Migrations/20240730083845_AddUniqueTermsVersionConstraint.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+namespace API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddUniqueTermsVersionConstraint : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateIndex(
+                name: "IX_Terms_Version",
```
Comment (ckr123) [inline domains/authorization/Authorization.API/API/Migrations/20240730083845_AddUniqueTermsVersionConstraint.cs:14]: Why index on version?

File: domains/authorization/Authorization.API/API/Migrations/20240730104826_AddDefaultTerms.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddDefaultTerms : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
```
Comment (ckr123) [inline domains/authorization/Authorization.API/API/Migrations/20240730104826_AddDefaultTerms.cs:14]: Why not use on migration file?

File: domains/authorization/Authorization.API/API/Migrations/20240730083845_AddUniqueTermsVersionConstraint.cs (hunk 1 above)
Comment (sahma19) [reply domains/authorization/Authorization.API/API/Migrations/20240730083845_AddUniqueTermsVersionConstraint.cs:14]: In order to enforce the unique constraint, it needs to look it up in the database, and creating an index for that, makes it more performant.

Comment (ckr123) [reply domains/authorization/Authorization.API/API/Migrations/20240730083845_AddUniqueTermsVersionConstraint.cs:14]: But i dont believe you need to make it an index to get the unique constraint? I dont think we get that many versions that you will notice any performance issues

File: domains/authorization/Authorization.API/API/Migrations/20240730104826_AddDefaultTerms.cs (hunk 2 above)
Comment (sahma19) [reply domains/authorization/Authorization.API/API/Migrations/20240730104826_AddDefaultTerms.cs]: Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1310/commits/99abc05de6b469ca9431a6920ed1dffb024aa2b3

**Review by ckr123 — APPROVED** [review]

**Pull request merged**